modules/nf-core/flash/main.nf

```diff
@@ -19,10 +19,12 @@ process FLASH {
 
     script:
     def args = task.ext.args ?: ''
+    def args2 = task.ext.args ?: ''
```

Comment:
What is this for?

Reply:
I inserted args2 to later add a --quiet function in the config file that I use in the pipeline.

Reply:
Maybe I can put everything in args1?

Reply:
Yes. args2 is used when multiple tools are being piped together (like bwa into samtools) so you can pass separate arguments to each tool ,or when tools have positional arguments (which isn't very common)

Reply:
Thank you for the review and for the clarity, I'll make the changes.